src/pages/about/index.mdx

```diff
@@ -22,73 +22,84 @@ keywords:
 
 import { Callout } from 'nextra/components'
 
-## About
+# Thinking Machines 
 
-We are **Thinking Machines**, a robotics company. We work on the intersection of humans and machines.
+![Eniac](./_assets/eniac.jpeg)
+_[Eniac](https://www.computerhistory.org/revolution/birth-of-the-computer/4/78), the World's First Computer (Photo courtesy of US Army)_
 
-At present, we are building the layer between AI and hardware, to turn your computer into an AI PC.
+## About 
 
-We like [open-source](https://github.com/janhq). We like [local-first](https://www.inkandswitch.com/local-first/). We love hardware. ❤️
+We're a team of AI researchers and engineers. We are the creators and lead maintainers of a few open-source AI tools: 
 
+- 👋 [Jan](https://jan.ai): ChatGPT-alternative that runs 100% offline
+- 🤖 [Cortex](/cortex): A simple, embeddable library to run LLMs locally
+- More to come! 
 
-## Our Work
+### Mission
 
-- [Jan](https://github.com/janhq/jan) is an open-source & self-hosted alternative to OpenAI's ChatGPT.
-- [Cortex](https://github.com/janhq/cortex) is a C++ backend that powers Jan. It is compatible with OpenAI API and multi-engine (llama.cpp, TensorRT-LLM).
-- [Open Source Models](https://huggingface.co/janhq): finetunes, datasets and more.
+We're a robotics company that focuses on the cognitive framework for future robots. Our long-term mission is to advance human-machine collaboration, to enable human civilization to thrive.
```

Comment:
We're a robotics company that focuses on the cognitive framework for future robots. Our long-term mission is to advance human-machine collaboration to enable human civilization to thrive.